projects/samples/environments/indoor/worlds/break_room.wbt

```diff
@@ -0,0 +1,1190 @@
+WorldInfo {
+  info [
+    "Copy of a real break room layout."
+    "This world contains the domestic objects included in Webots."
+    "Doors are interactive."
+    "This means that a robot could open a drawer, leave an object inside it, and close it."
+  ]
+  title "Break room layout"
+}
+Viewpoint {
+  orientation 0.5611620507608208 0.6084366364002342 0.5611613068199546 4.234793484858312
+  position -2.636688508521433 21.99638721155897 -0.12550161240670465
+}
+TexturedBackground {
+  texture "entrance_hall"
+  luminosity 0.5
+  skybox FALSE
+}
+TexturedBackgroundLight {
+  texture "entrance_hall"
+}
+Floor {
+  translation -0.93 0 -0.243
+  size 7.7 12.86
+  appearance PBRAppearance {
+    baseColorMap ImageTexture {
+      url [
+        "textures/light_strip_parquetry_base_color.jpg"
+      ]
+    }
+    roughness 1
+    roughnessMap ImageTexture {
+      url [
+        "textures/light_strip_parquetry_roughness.jpg"
+      ]
+    }
+    metalness 0
+    normalMap ImageTexture {
+      url [
+        "textures/light_strip_parquetry_normal.jpg"
+      ]
+    }
+    textureTransform TextureTransform {
+      rotation 1.57
+      scale 0.4 0.4
+    }
+  }
+}
+CeilingSpotLight {
+  translation 2.27 2 1.82
+  name "ceiling spot light"
+  spotLightColor 0.913725 0.72549 0.431373
+  spotLightIntensity 4
+  spotLightCastShadows TRUE
+}
+CeilingLight {
+  translation 0.87 2.5 -4.78
+  bulbColor 0.913725 0.72549 0.431373
+  supportColor 0.533333 0.541176 0.521569
+  pointLightColor 0.913725 0.72549 0.431373
+  pointLightIntensity 3
+  pointLightCastShadows TRUE
+}
+CeilingLight {
+  translation -2.74 2.5 -4.78
+  name "ceiling light(5)"
+  bulbColor 0.913725 0.72549 0.431373
+  supportColor 0.533333 0.541176 0.521569
+  pointLightColor 0.913725 0.72549 0.431373
+  pointLightIntensity 3
+  pointLightCastShadows TRUE
+}
+CeilingLight {
+  translation 0.87 2.5 -0.31
+  name "ceiling light(1)"
+  bulbColor 0.913725 0.72549 0.431373
+  supportColor 0.533333 0.541176 0.521569
+  pointLightColor 0.913725 0.72549 0.431373
+  pointLightIntensity 3
+  pointLightCastShadows TRUE
+}
+CeilingLight {
+  translation -2.74 2.5 -0.31
+  name "ceiling light(4)"
+  bulbColor 0.913725 0.72549 0.431373
+  supportColor 0.533333 0.541176 0.521569
+  pointLightColor 0.913725 0.72549 0.431373
+  pointLightIntensity 3
+  pointLightCastShadows TRUE
+}
+CeilingLight {
+  translation 0.87 2.5 3.52
+  name "ceiling light(2)"
+  bulbColor 0.913725 0.72549 0.431373
+  supportColor 0.533333 0.541176 0.521569
+  pointLightColor 0.913725 0.72549 0.431373
+  pointLightIntensity 3
+  pointLightCastShadows TRUE
+}
+CeilingLight {
+  translation -2.74 2.5 3.52
+  name "ceiling light(3)"
+  bulbColor 0.913725 0.72549 0.431373
+  supportColor 0.533333 0.541176 0.521569
+  pointLightColor 0.913725 0.72549 0.431373
+  pointLightIntensity 3
+  pointLightCastShadows TRUE
+}
+Wall {
+  translation 0 0 2.65
+  size 5 0.62 0.05
+  appearance Roughcast {
+    colorOverride 0.890196 0.803922 1
```

Comment:
This color looks a bit strange, is this wanted?

Reply:
Normally yes, I tried to reproduce as well as possible the colors I see in the videos attached for the set. It is possible that due to my graphics card, what I see is not what you see.